frontend/javascripts/admin/team/team_list_view.tsx

```diff
@@ -49,6 +51,35 @@ export function renderTeamRolesAndPermissionsForUser(user: APIUser) {
   ));
 }
 
+export function filterTeamMembersOf(team: APITeam, user: APIUser): boolean {
+  return (
+    user.teams.some((userTeam: APITeamMembership) => userTeam.id === team.id) ||
+    (user.isAdmin && user.isActive)
+  );
+}
+
+export function renderUsersForTeam(
+  team: APITeam,
+  allUsers: APIUser[] | null,
+  renderAdditionalContent = Function(),
```

Comment:
The `Function()` part is quite unusual in JS. Usually, it would be used to create a function from a string. Also `new Function()` would be cleaner. To avoid `Function()`, I would suggest this:

```suggestion
  renderAdditionalContent = (_teamMember: APIUser, _team: APITeam) => {},
```

Reply:
Alright! This is a good reminder to double check what I'm reading on blogs :sweat_smile:

Reply:
Why are you using the `_`syntax for `_teamMember` and `_team`?

Reply:
Makes sense, thanks!